docs/intro/installation.rst

```diff
@@ -253,35 +251,31 @@ touch if you want to help.
 
 - **Step 2**: Install Git
 
-  - In the ``Anaconda prompt``, run::
-
-       conda install -c conda-forge git
-
-    Note: Is has to be from ``conda-forge``, the anaconda version does not
-    provide the ``cp`` command.
+  - If you haven't installed :term:`Git` yet, please download and install the latest release from `git-scm.com/ <https://git-scm.com/>`_.
```

Comment:
Just FTR: Many options are advertised by the git installer. We should leave recommendations, if we have any. Candidates are:

- true type fonts (OMG my eyes can see for the first time on windows)
- msys terminal (stellar C&P performance in comparison)
- credential manager
- FS tuning options (do they have an effect?)
- enable symlinks?

Reply:
We need to automate, https://github.com/datalad/datalad/issues/5078